### PR TITLE
Code refactoring

### LINKED ISSUE #12379: Coding standard of function/method argument
See this https://doc.servo.org/script/dom/blob/struct.Blob.html for example, the `typeString` appears a lot and it is not obvious to me whether or not writing it as `type_string` would be proper.

As I understand, like DOM method bindings which doesn't follow common Rust naming convention, the arguments naming might share similar issue. But [coding standard](https://github.com/servo/servo/wiki/Coding-standards) doesn't specify this, nor was I ever being suggested to prefer certain style during the (short) contributing experience. So it is somewhat puzzling to me.

### PR DESCRIPTION
**Pull request by bubbles231**
The code was refactored to follow Rust naming conventions better.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes help fix #12379.

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because logic was not changed.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13307)

### CODE REVIEW

File: components/script/dom/htmliframeelement.rs [hunk 1]
```diff
@@ -402,7 +402,7 @@ unsafe fn build_mozbrowser_event_detail(event: MozBrowserEvent,
     }
 }
 
-pub fn Navigate(iframe: &HTMLIFrameElement, direction: TraversalDirection) -> ErrorResult {
```
Comment (Manishearth) [inline components/script/dom/htmliframeelement.rs]: This shouldn't be renamed

File: components/script/dom/htmliframeelement.rs [hunk 2]
```diff
@@ -450,7 +450,7 @@ impl HTMLIFrameElementMethods for HTMLIFrameElement {
             let self_url = self.get_url();
             let win_url = window_from_node(self).get_url();
 
-            if UrlHelper::SameOrigin(&self_url, &win_url) {
```
Comment (Manishearth) [inline components/script/dom/htmliframeelement.rs]: Ditto

File: components/script/dom/location.rs [hunk 3]
```diff
@@ -62,45 +62,45 @@ impl LocationMethods for Location {
 
     // https://html.spec.whatwg.org/multipage/#dom-location-hash
     fn Hash(&self) -> USVString {
-        UrlHelper::Hash(&self.get_url())
+        UrlHelper::hash(&self.get_url())
```
Comment (Manishearth) [inline components/script/dom/location.rs]: I think these should keep the capitalization

File: components/script/dom/urlhelper.rs [hunk 4]
```diff
@@ -10,25 +10,25 @@ use url::{Url, quirks};
 pub struct UrlHelper;
 
 impl UrlHelper {
```
Comment (Manishearth) [inline components/script/dom/urlhelper.rs]: These should keep the names

File: components/script/dom/window.rs [hunk 5]
```diff
@@ -1536,7 +1536,7 @@ impl Window {
     }
 
     // https://html.spec.whatwg.org/multipage/#accessing-other-browsing-contexts
-    pub fn IndexedGetter(&self, _index: u32, _found: &mut bool) -> Option<Root<Window>> {
```
Comment (Manishearth) [inline components/script/dom/window.rs]: This shouldn't be changed

**Review by Manishearth — CHANGES_REQUESTED** [review]
We shouldn't be renaming `CamelCase` (not `camelCase`) methods on DOM types.

The `URLHelper` methods are a bit of an edge case here -- they're not directly called from Javascript, but I think they're supposed to mirror the URL interface.

**Review by Manishearth — APPROVED** [review]
Looks good to me!

**Comment by bubbles231** [comment]
I am excited to make my first PR to servo. It is really close -- There is only one out of 9977 tests that are failing. 
````
Tests with unexpected results:
  ▶ TIMEOUT [expected PASS] /css21_dev/html4/height-072.htm
  │ 
  │ Xlib:  extension "XFree86-VidModeExtension" missing on display ":0".
  │ failed to receive response to font request: IoError(Error { repr: Os { code: 104, message: "Connection reset by peer" } }) (thread LayoutWorker worker 1/12, at ../src/libcore/result.rs:799)
  │ stack backtrace:

````

Can you help me understand how to re-run this one failing test?

It seems like the error might be related to font selection in CSS. Do you have an idea of where in the
code that issue might be coming from?

**Comment by Manishearth** [comment]
@bors-servo retry

Looks like an intermittent (let's see if it really is)

**Pull request merged**